modules/core/src/test/java/org/apache/ignite/internal/processors/service/IgniteServiceConfigVariationsFullApiTest.java

```diff
@@ -144,6 +144,7 @@ public void testKeyAffinityDeploy() throws Exception {
      */
     @Test
     public void testMultipleDeploy() throws Exception {
+        System.out.println("~~! " + testsCfg.description());
```

Comment:
I suggest to use `GridAbstractTest.info` here (if this output is really needed).

Reply:
@pavlukhin No, it is just to debug in the context of `testsCfg` question.
I'll remove it in the next commit.